src/main/webapp/2Dscript.js

```diff
@@ -1,24 +1,43 @@
-// Gobal variables.
+// Global variables.
 let map;
-let maps = {}; // Cache for map elements that have been loaded.
-let datas = {}; // Cache for data elements that have been loaded.
+const mapCache = {};
+const dataCache = {};
 let pose;
 let data;
 let id;
 let type;
 let currentLat;
 let currentLong;
-// Define bounding box around the different data sets to quickly determined
-// whether they should be included in the calculation or not
+let initialPoseData;
+
+/**
+ * A Pose data object containing all properties returned
+ * from the requisite proto in the JSON format.
+ * Position is given in WGS84 latitude, longitude, and altitude.
+ * @typedef {Object<number, number, number, number,
+ * number, number, number>} PoseData
+ * @property {number} gpsTimestamp The time in seconds.
+ * @property {number} lat Degrees in [-90, 90]
+ * @property {number} lng Degrees in [-180, 180]
+ * @property {number} alt Meters above WGS84
+ * @property {number} rollDeg Degrees in [0,360]
+ * @property {number} pitchDeg Degrees in [0,360]
```

Comment:
I think pitch is actually in [-90, 90].

Reply:
Copied this from the master branch proto comment, so would that mean it was wrong there too?

Reply:
Probably so.